### DISCUSSION TITLE
iTwin github discussion  may have been hacked?

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#4856] [General] [answered]
To whomever is the administrator of this iTwin git repository:

A few hours ago I received the following email related to an iTwin discussion on git hub that I started.  The link in Patricia Johnson's reply takes you to a sex site.  I don't know Patricia but I am assuming her email was hacked:

![image](https://user-images.githubusercontent.com/30668739/209988129-c802a319-0b63-4a61-8394-4adf12a37cbf.png)

**Comment by calebmshafer** [comment]
@JohnMajerle thanks for bringing this to our attention. This has happened on a few of our discussions in the past 24 hours and the post has been taken down, however the emails were still delivered. We will report these to GitHub and see what they're planning to do in order to prevent this from happening.

**Reply by JohnMajerle** [reply]
Thanks Caleb.  Hopefully no one is getting burned by it.  Hope you and all my old Bentley friends had a nice holiday!

**Comment by JohnMajerle** [comment]
Additional information:  I am getting similar hacked email notifications from other iTwin github issues I am subscribed to.

**Comment by pmconne** [comment, marked as answer]
This issue plagued many, many GitHub repos earlier today - not just iTwin. I reported (and deleted) a few in itwinjs-core. GitHub team appears to be on top of it.

EDIT: [discussed on HN](https://news.ycombinator.com/item?id=34167965).
